backend/common/constants.py

```diff
@@ -1 +1,2 @@
 POKEAPI_BASE_URL = "https://pokeapi.co/api/v2/"
+POKEAPI_POKEMONS_LIMIT = 1118
```

Comment:
You shouldn't need this. Your code should work regardless of the number of existing pokemons in the API. What if new ones are included?

Reply:
Hmm, that makes sense.